source/solvers/postprocessing_cfd.cc

```diff
@@ -1040,3 +1040,81 @@ calculate_flow_rate(const DoFHandler<3> &                     dof_handler,
                     const unsigned int &                      boundary_id,
                     const Quadrature<2> &face_quadrature_formula,
                     const Mapping<3> &   mapping);
+
+template <int dim, typename VectorType>
+std::pair<double, double>
+calculate_min_max_temperature(const DoFHandler<dim> &dof_handler_ht,
+                              const VectorType &     evaluation_point,
+                              const Quadrature<dim> &quadrature_formula,
+                              const Mapping<dim> &   mapping)
+{
+  const FESystem<dim, dim> fe = dof_handler_ht.get_fe();
+
+  const unsigned int  n_q_points = quadrature_formula.size();
+  std::vector<double> local_temperature_values(n_q_points);
+
+  FEValues<dim> fe_values(mapping,
+                          fe,
+                          quadrature_formula,
+                          update_values | update_quadrature_points |
+                            update_JxW_values);
```

Comment:
Why update the quadrature points and the JxW values, you only use the values anyway? Might as well. update as minimally as possible.